streams/src/main/java/org/apache/kafka/streams/processor/StateStore.java

```diff
@@ -58,4 +58,10 @@
      * @return  {@code true} if the storage is persistent&mdash;{@code false} otherwise
      */
     boolean persistent();
```

Comment:
I think it's important to maintain clarity independently of conventions. `getName()` and `name()` are both clear and the former has additional verbosity. `open` is quite ambiguous IMO. If I saw a method name like that, I would expect it to `open` something instead of returning whether it is open. I am not sure if the `is` prefix is discouraged in the same way as the `get` prefix for this reason.